heralding/capabilities/imap.py

```diff
@@ -0,0 +1,167 @@
+# Copyright (C) 2017 Johnny Vestergaard <jkv@unixcluster.dk>
```

Comment:
I tried to follow Heralding general style of coding, so I based Imap class on Pop3 class. 
Thus I think, it is still yours copyright.